### DISCUSSION TITLE
Response Timing & Eufy Sensor for Light Automation

### DISCUSSION THREAD

**Discussion by Mwalton4uk** [fuatakgun/eufy_security#1029] [Q&A] [answered]
Hi all, just wondering, how fast would you say the update from Eufy to HA is? I am wanting to add a motion sensor in my hallway to do 2 things:

1. Turn on the lights
2. When in away mode, sound alarm

My options are using a Zigbee or WiFi Motion sensor and triggering alarm through the trigger alarm button OR using a Eufy Sensor and triggering the lights through when the motion detected sensor changes and I am just wondering, how quick are is this integration in updating?

**Comment by fuatakgun** [comment, marked as answer]
Don't spend money on this closed ecosystem if you want to use it inside your home assistant. Eufy is great if you stick with their app and their products but anytime in the future, this integration might stop working


Speed wise, integration relies on push notifications taking 1 to 3 seconds for me, when it is working :-)

**Reply by Mwalton4uk** [reply]
Fair point, hadn't considered that angle!

If i imagine this integration stops working tomorrow, I guess it depends on which feature I want more, Motion Sensor for Light or for Alarm. Wish Eufy would open up the system as other than it being semi-closed, it's almost perfect compared to other solutions.

Anyway, thanks for input :D Ill prob get something like Aqara P1 and just use automation to trigger alarm.
